pkg/cmd/unpack/unpack.go

```diff
@@ -221,7 +220,11 @@ func unpackLayer(ctx context.Context, store content.Storage, desc ocispec.Descri
 		return fmt.Errorf("failed to create directory %s: %w", unpackDir, err)
```

Comment:
Not related to these changes but on line 213 `cErr` is checked for nil but on the `err` is used for return value.

Reply:
Good catch! Fixed.